.github/workflows/scripts/capture-log.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "📄 Starting log capture process..."
+LOG_DIR="logs"
```

Comment:
```suggestion
LOG_DIR="log"
```

Might as well piggy back off of the directory with logs in already?

Reply:
Where is that?

Reply:
I was assuming we’d checked out the code so in the repo already